### PR TITLE
Test get_dandiset_stats

### LINKED ISSUE #374: Add backend tests for get_dandiset_stats
The `get_dandiset_stats` endpoint needs tests. It should be as simple as creating a dandiset and verifying that the stats increment correctly when adding folders and items of various sizes in various combinations.

### PR DESCRIPTION
**Pull request by jwodder**
Closes #374

### CODE REVIEW

File: girder-dandi-archive/tests/rest/test_get_dandiset_stats.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                "name": name,
+                "mimeType": "text/plain",
+                "size": len(content),
+            },
+            body=content,
+            type="text/plain",
+            isJson=False,
+            **request_auth,
+        )
+        assertStatusOk(resp)
```
Comment (dchiquit) [inline girder-dandi-archive/tests/rest/test_get_dandiset_stats.py:107]: These helpers would be useful in the other test methods. Can you move these to the top of the file and invoke them in the other test methods?

Comment (dchiquit) [inline girder-dandi-archive/tests/rest/test_get_dandiset_stats.py:107]: These helpers would be useful in the other test methods. Can you move these to the top of the file and invoke them in the other test methods?

Comment (jwodder) [reply girder-dandi-archive/tests/rest/test_get_dandiset_stats.py]: Done.

**Comment by mgrauer** [comment]
@jwodder  same issue here if you don't mind, issuing the PR from this repo. Thanks and sorry for the bother.

**Comment by yarikoptic** [comment]
FWIW, added @jwodder into dandiarchive team to be able to submit through branches in this repo

**Comment by jwodder** [comment]
@mgrauer Resubmitted as #466.

**Pull request closed without merging**